### ISSUE TITLE
Unified Menu Controls & Menu Toggle Hotkey not working for secondary gamepads

### ISSUE THREAD

**Issue by ner00** [libretro/RetroArch#10032]
## Description

When Unified Menu Controls (UMC) are enabled, Menu Toggle Combo Hotkey (MTCH) for gamepad other than first can exit the quick menu but cannot enter it.

### Expected behavior

When UMC and MTCH are enabled, all controllers should be able to toggle the menu.

### Actual behavior

Only the first controller can enter/exit the quick menu, other controllers can only toggle the menu off but not on.

### Steps to reproduce the bug

1. Press MTCH in second gamepad - *nothing happens*.
2. Press MTCH in first gamepad - *enters quick menu*.
3. Press MTCH in second gamepad - *quits quick menu*.

### Bisect Results

Unsure, it is the first time I have tried toggling the menu from another controller other than Port #0.

### Version/Commit

- RetroArch: v1.8.3 (Jan 10 2020/8032ff6e19)

### Environment information

- OS: Windows 10 x64 Build 18362 (v10.0)

**Comment by cmitu** [comment]
Looks like a duplicate of https://github.com/libretro/RetroArch/issues/3337 ?

**Comment by ner00** [comment]
Not really, that one is mostly complaining about the control of the menu, which works fine in this case. One of the last comments hints to this issue (https://github.com/libretro/RetroArch/issues/3337#issuecomment-321132214) but it is not focused on it particularly.

**Comment by zoltanvb** [comment]
Tested now, it works in current version, menu toggle combo works also in port 2 with/without unified menu controls.
